app/src/main/java/com/example/cinemapp/ui/main/SearchFragment.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.cinemapp.ui.main
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.cinemapp.R
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [SearchFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class SearchFragment : Fragment() {
+    // TODO: Rename and change types of parameters
+    private var param1: String? = null
```

Comment:
Remove unused parameter and onCreate method, since this is generated by Android studio, and it is not useful to our implementation.